GitVersion.yml

```diff
@@ -1,6 +1,7 @@
 assembly-versioning-scheme: Major
-next-version: 1.0.0
-commit-message-incrementing: Disabled
+next-version: 1.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```

Comment:
What's the reason this removed? Just to drop `rc` from the packages built with `release-x` branches?